### PR TITLE
add relativePathAsNamespace option

### PR DESCRIPTION
**Pull request by smccollum-blurb**
The only thing missing from this loader is the ability to load a file format like this, so I added it. It's a new option so it shouldn't interfere with anything else, and it introduced no new dependencies, so I feel like it's very safe. Let me know if there's anything else I can do to help get this in. Thanks! ^__^

### CODE REVIEW

**Comment by rosskevin** [comment]
I'll take a look at the build error

File: test/relativepath.spec.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    context(type, () => {
+      let thisScope;
+      beforeEach(function (done) {
+        //mock webpack loader this scope
+        const emptFn = function () {}
+        thisScope = {
+          addDependency: emptFn,
+          addContextDependency: emptFn,
+          cacheable: emptFn,
+          resource: path.join(__dirname, `./data/basic-app-context-${type}/locales/index.js`),
```
Comment (rosskevin) [inline test/relativepath.spec.js]: Let's go ahead and rename this directory from `basic-app-context-` to the option name prefix `relativePathAsNamespace-`

**Review by rosskevin — DISMISSED** [review]
I'll get the build running, if you can make this one change it looks good to go.

**Comment by rosskevin** [comment]
@smccollum-blurb the ci issue is fixed, next push should build properly.

**Comment by rosskevin** [comment]
Thanks, autoreleased as @alienfast/i18next-loader@1.0.18

**Comment by smccollum-blurb** [comment]
Sweeeet! Thanks a bunch!

**Pull request merged**